PoorMansTSqlFormatterLibShared/Formatters/TSqlStandardFormatter.cs

```diff
@@ -562,6 +563,13 @@ private void ProcessSqlNode(Node contentElement, TSqlStandardFormattingState sta
                                 break;
                               }
                               break;
+                            case SqlStructureConstants.ENAME_FUNCTION_PARENS: // This stops the inline functions from getting a line break
+                              switch (contentElement.Name)
+                              {
+                                case SqlStructureConstants.ENAME_EXPRESSION_PARENS:
+                                  break;
+                              }

```

Comment:
Does this section do anything?

The outer switch statement is on `contentElement.Parent.Name`, and this inner one is on `contentElement.Name`.

The new case for `SqlStructureConstants.ENAME_FUNCTION_PARENS` catches something that would have otherwise fallen through to the `default` case, but the inner switch statement doesn't seem like it does anything.

Reply:
I remember that. Probably an errant test artifact that I had breakpoints on..... Debugging this code is a bear

Reply:
Yeah, I looked through this code for the first time as part of this PR, and... wow, what a switch statement. Thanks for working on this 😄

Reply:
I removed the switch case to nowhere and added a comment on why the empty case is present